main.go

```diff
@@ -5,58 +5,145 @@ import (
 	"log"
 
 	"github.com/sensu-community/sensu-plugin-sdk/sensu"
-	"github.com/sensu/sensu-go/types"
+	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	wavefront "github.com/wavefronthq/wavefront-sdk-go/senders"
 )
 
-type HandlerConfig struct {
+// Config represents a handler config for the wavefront handler.
+type Config struct {
 	sensu.PluginConfig
-	Example string
+	Host                 string
+	MetricsPort          int
+	FlushIntervalSeconds int
+	Prefix               string
+	Tags                 map[string]string
 }
 
-type ConfigOptions struct {
+// Options represents the config options for the wavefront handler.
+type Options struct {
 	Example sensu.PluginConfigOption
 }
 
+const (
+	host   = "host"
+	port   = "metrics-port"
```

Comment:
There are multiple ports that can be configured for a wavefront proxy sender (see https://github.com/wavefrontHQ/wavefront-sdk-go#proxy-sender). I'm making the distinction that this is a metrics port rather than a distribution or tracing port.